config/neodymium.properties

```diff
@@ -196,6 +196,10 @@ neodymium.webDriver.firefox.legacyMode = false
 ## Whether to use a single driver instance for all tests (defaults to false).
 neodymium.webDriver.reuseDriver = false
 
+## The number of reuses. If not specified or set below 1 the driver will be reused unlimited times.
+## Setting the property to 1 means that the driver is reused once but twice in total.
```

Comment:
```suggestion
## Setting the property to 1 means that the driver is reused once, so used twice in total.
```
Otherwise it sounds like it is reused twice in total.